concent_api/core/migrations/0003_auto_20180109_1436.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-01-09 14:36
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def migrate_golem_messages_types_from_strings_to_ints(apps, schema_editor):
+
+    Message = apps.get_model('core', 'Message')
+
+    for message in Message.objects.all():
+        if message.type == 'ForceReportComputedTask':
+            message.type = 4001
+        if message.type == 'AckReportComputedTask':
+            message.type = 4002
+        if message.type == 'RejectReportComputedTask':
+            message.type = 4003
+        if message.type == 'VerdictReportComputedTask':
+            message.type = 4004
+        message.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0002_auto_20180105_0945'),
+    ]
+
+    operations = [
+        migrations.RunPython(migrate_golem_messages_types_from_strings_to_ints),
+        migrations.AlterField(
+            model_name='message',
+            name='type',
+            field=models.PositiveSmallIntegerField(),
+        ),
```

Comment:
Does this even work? Please test this migration.

Your migration reads a string from a database column then stores a number in the same column and saves. This cannot work. The column would have to magically change its type during the operation.

This cannot be done in less than three separate operations, four if the new column must have the same name as the old one:
- Add an integer column.
- Fill the new column with values based on the string column.
- Remove the string column.
- Rename the integer column so that has the name string column used to have.